ethcore/res/ethereum/classic.json

```diff
@@ -41,11 +41,13 @@
 		"eip145Transition": "0x921288",
 		"eip1014Transition": "0x921288",
 		"eip1052Transition": "0x921288",
-		"eip1283Transition": "0xa03ae7",
+		"eip1283Transition": "0x0",
+		"eip1283DisableTransition": "0x0",
+		"eip1283ReenableTransition": "0xa03ae7",
```

Comment:
Is there a better way to handle this without three flags?

Reply:
You can just use `eip1283Transition`.

Reply:
If you want to keep https://github.com/openethereum/openethereum/pull/11598/files#diff-53fd86a31b752879ee69bd6f13bb729bR25-R28, then no.

Reply:
#11305